app/views/posts/index.html.erb

```diff
@@ -1,2 +1,18 @@
-<h1>Posts#index</h1>
```

Comment:
Please follow [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAAB34BLWT5JLXGA5IRSWAYZXAY4JGBTQ) wireframe for posts index page. The layout of the page should be the same as in the given wirefram.